core/dir/empty_spec.rb

```diff
@@ -0,0 +1,37 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+require File.expand_path('../fixtures/common', __FILE__)
+
+describe "Dir.empty?" do
+  ruby_version_is "2.4" do
```

Comment:
Could you swap those two lines? It makes it clear that the method does not exist before Ruby 2.4.